scripts/rnnlm/get_special_symbol_opts.py

```diff
@@ -42,13 +42,19 @@
         if special_symbols[sym][0] != lower_ids[sym]:
             print('{0}={1} '.format(special_symbols[sym][1], lower_ids[sym]), end='')
             printed = True
```

Comment:
i'm unclear about why these changes were made to this file-- what issue were you trying to fix?
I suspect the correct fix here is just to remove an if-statement, because your new code is of the form:
```
if foo:
  bar
else
  bar
```